actionview/lib/action_view/rendering.rb

```diff
@@ -92,12 +92,16 @@ def rendered_format
       # Find and render a template based on the options given.
       # :api: private
       def _render_template(options) #:nodoc:
-        variant = options[:variant]
+        variant = options.delete(:variant)
+        assigns = options.delete(:assigns)
+        context = view_context
+
+        context.assign assigns if assigns
```

Comment:
Sure, I'll update `view_context` method to receive an optional value of `:assigns` and return a context with it.

I'd prefer to leave removal of `view_assigns` dependency from ActionView as a separate pull-request, if you don't mind.